### DISCUSSION TITLE
PCトーカーのユーザー向け情報

### DISCUSSION THREAD

**Discussion by nishimotz** [nvdajp/nvdajp#323] [お役立ち情報]
NVDA日本語チームとして、PCトーカーのユーザーに向けて
NVDAを併用してもらうための情報を整備する、という課題を、
このチケットで議論します。

2022年2月8日追記：本記事を issue から discussion に移行しました。

**Comment by TetsuroOkane** [comment]
以下は今までのサポートの中でお問い合わせの多かったものをまとめたものです。

### PC-Talkerを含め、他のスクリーンリーダーを併用する時は、必ず下記手順での切り替えを行う
• 他のスクリーンリーダーを終了してからNVDAを起動する
• NVDAを終了してから他のスクリーンリーダーを起動する

### NVDAとPC-Talkerの切り替え手順
PC-TalkerからNVDAに切り替える
• 終了：Ctrl+Alt+F3を押し、「PC-Talker 終了」と読まれた事を確認してください。
• Ctrl+Alt+Nを押してNVDAを起動します。
### 【重要】PC-Talkerの起動と終了を正しく行う
誤ってCtrl+Altを押しながらF3の隣のF2を押してしまった場合やPC-Talkerを起動したままCtrl+Alt+NでNVDAを起動した時に音声停止状態でPC-Talkerが起動したままになり、NVDAのコマンドが動作しないなどの不具合を生じますので、必ず先にCtrl+Alt+F3を押した後「PC-Talker 終了」と読まれるのを確認してからNVDAを起動してください。

### NVDAからPC-Talkerに切り替える
• 終了：Insertまたは無変換+Qを押してください。初期状態では終了ダイアログが開いたら、エンターキーを押してください。下がり調子のメロディーが鳴りNVDAが終了します。
※NVDA設定から「NVDA終了時に終了オプションを表示」のチェックを外しておくと、NVDA+QのみでNVDAが終了し、スムーズに切り替えられます。
• 起動：Ctrl+Shift+F3を押してください。「PC-Talkerを組み込みました」とガイダンスされれば切り替え完了です。

### PC-Talkerの起動と終了コマンド
• 起動：Ctrl+Shift+F3
• 終了：Ctrl+Alt+F3

### New AOKシステム関連の自動起動アプリの停止
PC-Talkerを終了していてもNVDAが正常に動作しない場合、「マイスタートメニュー」や「AOKメニュー」が起動している可能性があります。
「マイサポート」から「ツール」の中の「PC-Talkerの初期設定」にある「自動起動の設定」を選び、「AOKメニュー」と「マイスタートメニュー」の状態を確認して、チェックが入っていた場合、チェックを外してください。

### KSDオフィスメニューの無効化
PC-Talkerのインストールされたパソコンで、WordやExcelを使用していてNVDAが正常に動作しない時は、アドインの中のKSDオフィスメニューを無効化しておく必要があります。
